python/README.md

```diff
@@ -50,7 +50,47 @@ object_name = 'file1'
 data = sc.textFile(cos.url(object_name, bucket_name))
 ```
 
-### Bluemix / Data Science Experience
+### Bluemix IBM Cloud Object Storage (COS)
+The class CloudObjectStorage allows you to connect to an IBM bluemix COS. You can connect to a bluemix COS using api keys
+as follows:
+
+```python
+import ibmos2spark
+
+credentials = {
+    'endpoint': 'XXX',
+    'api_key': 'XXX',
+    'service_id': 'XXX'
+}
+
+configuration_name = 'os_bluemix_cos_config'
+cos = ibmos2spark.CloudObjectStorage(sc, credentials, configuration_name, 'bluemix_cos')
+
+bucket_name = 'bucket_name'
+object_name = 'file_name'
+data = sc.textFile(cos.url(object_name, bucket_name))
+```
+
+Alternatively, you can connect to an IBM bluemix COS using IAM token. Example:
+```python
+import ibmos2spark
+
+credentials = {
+    'endpoint': 'XXX',
+    'iam_token': 'eyJraWQXXXX .... X',
+    'service_id': 'XXX'
+}
+
+configuration_name = 'os_bluemix_cos_config'
+cos = ibmos2spark.CloudObjectStorage(sc, credentials, configuration_name, 'bluemix_cos', 'iam_token')
+
+bucket_name = 'bucket_name'
+object_name = 'file_name'
+data = sc.textFile(cos.url(object_name, bucket_name))
```

Comment:
Having the bucket_name at the end is due to the fact that this param is optional and in python it has to go to the end.

Reply:
We could think about it if we want to change this in a next release

Reply:
Right, but it's only optional because when this was first supported, we used a fixed configuration name. I was opposed to that because it didn't support multiple configurations like the other implementations, but it was pushed ahead anyways and released too quickly, in my opinion.